database/factories/StudentFactory.php

```diff
@@ -22,7 +22,7 @@
     return [
         'user_id' => function () use ($studentNumber) {
             return factory(User::class)->create([
-                'email' => "${studentNumber}@alunos.uminho.pt",
+                'email' => "${studentNumber}@".config('app.mail_domain'),
```

Comment:
Hum, I suggest:
```suggestion
                'email' => "${studentNumber}@.${config('app.mail_domain')}",
```
or

```suggestion
                'email' => 'studentNumber@'.config('app.mail_domain'),
```

Use either string interpolation or concatenation 😅